docs/using-the-nodejs-wrapper/UsingTheNodejsWrapper.md

```diff
@@ -50,7 +50,7 @@ These parameters are applicable to any instance of the AWS Advanced NodeJS Wrapp
-| `mysqlQueryTimeout`            | `number`  | No                                                                               | Query timeout in milliseconds. This is only applicable when using the AwsMySQLClient. To set query timeout for the AwsPGClient, please use the built-in `query_timeout` parameter. See the `node-postgres` [documentation](https://node-postgres.com/apis/client) for more details..                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | 20000                                                                                                                                                                        |
+| ~~`mysqlQueryTimeout`~~        | `number`  | No                                                                               | Query timeout in milliseconds. This is only applicable when using the AwsMySQLClient. To set query timeout for the AwsPGClient, please use the built-in `query_timeout` parameter. See the `node-postgres` [documentation](https://node-postgres.com/apis/client) for more details..                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | 20000                                                                                                                                                                        |
```

Comment:
Lets also mention that this is deprecated and that applications should opt for wrapperQueryTimeout instead.
Also there is an extra full-stop at the end.